packages/charts/frontend/sirius-components-charts/src/pie/PieChart.tsx

```diff
@@ -84,7 +84,8 @@ export const PieChart = ({ width, height, chart }: PieChartProps) => {
       .attr('width', width)
       .attr('height', height)
       .attr('viewBox', [-width / 2, -height / 2, width, height])
-      .attr('style', 'max-width: 100%; height: auto; height: intrinsic;');
+      .attr('style', 'max-width: 100%; height: auto; height: intrinsic;')
+      .attr('pointer-events', 'none'); // allow selection anywhere in the svg
```

Comment:
Fix selection problem when users were trying to select the pie chart by clicking in its colored part.